flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -909,7 +909,7 @@ def create_resume_file_names_map(
                 liketype=liketype,
             )
             input_file_name = output_file_name
-            if os.environ.get("FLEPI_BLOCK_INDEX") == "1":
+            if flepi_block_index == "1":
```

Comment:
i assume this is just fixing a random error, that you happened to turn up fixing this issue broadly?

Reply:
yes, I  found this parameter is not used to replace the reading of the environmental variable. So I added this tiny fix.